### ISSUE TITLE
Can't use /react monitor in 1.13

### ISSUE THREAD

**Issue by CaptainAmpersand** [VolmitSoftware/React#438]
```[01:35:09 INFO]: Ampersand issued server command: /react monitor
[01:35:09 ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'react' in plugin React v6.569
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:46) ~[server.jar:git-Spigot-ed1cec9-4e2f571]
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:141) ~[server.jar:git-Spigot-ed1cec9-4e2f571]
        at org.bukkit.craftbukkit.v1_13_R1.CraftServer.dispatchCommand(CraftServer.java:692) ~[server.jar:git-Spigot-ed1cec9-4e2f571]
        at net.minecraft.server.v1_13_R1.PlayerConnection.handleCommand(PlayerConnection.java:1634) ~[server.jar:git-Spigot-ed1cec9-4e2f571]
        at net.minecraft.server.v1_13_R1.PlayerConnection.a(PlayerConnection.java:1469) ~[server.jar:git-Spigot-ed1cec9-4e2f571]
        at net.minecraft.server.v1_13_R1.PacketPlayInChat.a(PacketPlayInChat.java:45) ~[server.jar:git-Spigot-ed1cec9-4e2f571]
        at net.minecraft.server.v1_13_R1.PacketPlayInChat.a(PacketPlayInChat.java:1) ~[server.jar:git-Spigot-ed1cec9-4e2f571]
        at net.minecraft.server.v1_13_R1.PlayerConnectionUtils.a(SourceFile:10) ~[server.jar:git-Spigot-ed1cec9-4e2f571]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_171]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_171]
        at net.minecraft.server.v1_13_R1.SystemUtils.a(SourceFile:198) [server.jar:git-Spigot-ed1cec9-4e2f571]
        at net.minecraft.server.v1_13_R1.MinecraftServer.w(MinecraftServer.java:884) [server.jar:git-Spigot-ed1cec9-4e2f571]
        at net.minecraft.server.v1_13_R1.DedicatedServer.w(DedicatedServer.java:411) [server.jar:git-Spigot-ed1cec9-4e2f571]
        at net.minecraft.server.v1_13_R1.MinecraftServer.v(MinecraftServer.java:819) [server.jar:git-Spigot-ed1cec9-4e2f571]
        at net.minecraft.server.v1_13_R1.MinecraftServer.run(MinecraftServer.java:717) [server.jar:git-Spigot-ed1cec9-4e2f571]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_171]
Caused by: java.lang.IllegalArgumentException: No enum constant org.bukkit.Sound.ENDERDRAGON_HIT
        at java.lang.Enum.valueOf(Enum.java:238) ~[?:1.8.0_171]
        at org.bukkit.Sound.valueOf(Sound.java:1) ~[server.jar:git-Spigot-ed1cec9-4e2f571]
        at com.volmit.react.util.MSound.bukkitSound(MSound.java:229) ~[?:?]
        at com.volmit.react.Gate.msgError(Gate.java:749) ~[?:?]
        at com.volmit.react.api.Capability.sendNotCapable(Capability.java:157) ~[?:?]
        at com.volmit.react.command.CommandMonitor.fire(CommandMonitor.java:47) ~[?:?]
        at com.volmit.react.controller.CommandController.onCommand(CommandController.java:311) ~[?:?]
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:44) ~[server.jar:git-Spigot-ed1cec9-4e2f571]
        ... 15 more
```
Spigot 1.13 (First release)

**Comment by EthanTG** [comment]
![react doesn t support 1 13](https://user-images.githubusercontent.com/29691919/43091080-330d4b02-8e77-11e8-8c56-6d56f57278e1.png)

**Comment by CaptainAmpersand** [comment]
![image](https://user-images.githubusercontent.com/33905976/43102293-91ee9614-8e7f-11e8-9f01-3d5edba5d94c.png)

**Comment by cyberpwnn** [comment]
Implemented in React 6.572
